package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-oauth-flow",
+  "name": "react-oauth-flow-coffreo",
```

Comment:
that was to prevent confusion for the user when he tape `yarn lint "react-oauth-flow" `. 
This way he is sure he link this package and not the official one.

Reply:
`yarn link` ? no ?

Reply:
Thanks for feedback.

I can another advantage with this update : we can envisage to publish this package to npm repository too ;)